MekHQ/src/mekhq/gui/dialog/MassMothballDialog.java

```diff
@@ -106,14 +102,16 @@ public MassMothballDialog(Frame parent, Unit[] units, Campaign campaign, boolean
         gbc.weightx = 3;
         JLabel instructionLabel = new JLabel();
         instructionLabel.setBorder(new LineBorder(Color.BLUE));
-        instructionLabel.setText("<html>Choose the techs to carry out mothball/reactivation operations on the displayed units. <br/>A * indicates that the tech is currently maintaining units.</html>");
```

Comment:
While you're in this file, there's a bug in this dialog that shows activation time cost to be the same as mothball time cost, where it should be 1/2 the time. Do you think you can track it down?

Reply:
I'll take a look, and I've actually got an idea of where it is.

Reply:
Fixed, and found another bug that is somewhat related that I fixed at the same time.